From 72b84ab038b45487c389bb124d7f7f413a099b89 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Thu, 22 Dec 2016 17:28:33 +0100
Subject: [PATCH 171/370] arm: include definition for cpumask_t
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.9/older/patches-4.9.146-rt125.tar.xz

This definition gets pulled in by other files. With the (later) split of
RCU and spinlock.h it won't compile anymore.
The split is done in ("rbtree: don't include the rcu header").

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
[carnil: Refresh for context changes in 4.9.165 due to 625655727eb5
("ARM: 8824/1: fix a migrating irq bug when hotplug cpu")]
---
 arch/arm/include/asm/irq.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/include/asm/irq.h
+++ b/arch/arm/include/asm/irq.h
@@ -22,6 +22,8 @@
 #endif
 
 #ifndef __ASSEMBLY__
+#include <linux/cpumask.h>
+
 struct irqaction;
 struct pt_regs;
 
